src/layout/CTCookieAgreement/index.js

```diff
@@ -0,0 +1,102 @@
+import React, {useState} from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import { Link } from 'react-router-dom';
+import { SignInMenu } from '../CTNavHeader/NavHeaderMenu/SignInMenu';
+import './index.scss';
+
+export const AGREEMENT_ACCEPTED_KEY = 'class_transcribe_agreement_accepted';
+
+function CTCookieAgreement() {
+  // Set true to false for testing purposes
+  const [dialogOpen, setDialogOpen] = useState(!(localStorage.getItem(AGREEMENT_ACCEPTED_KEY) === 'true'));
+  const [anchorEl, setAnchorEl] = useState(null);
+
+  const handleAcceptLoginClick = (e) => {
+    localStorage.setItem(AGREEMENT_ACCEPTED_KEY, 'true');
+    setAnchorEl(e.currentTarget);
+  };
+  
+  const handleAcceptLoginClose = () => {
+    setTimeout(() => setAnchorEl(null), 200);
+  };
+
+  const handleCloseBrowserWindow = () => {
+    window.open('about:blank', '_self').close();
+  }
+
+  const handleAcceptSkipLogin = () => {
+    setDialogOpen(false);
+    localStorage.setItem(AGREEMENT_ACCEPTED_KEY, 'true');
+  }
+
+  return (
+    <Dialog
+      open={dialogOpen} 
+      disableBackdropClick
+      disableEscapeKeyDown
+      fullWidth
+      maxWidth='sm'
+    >
+      <div id='cookie-agreement-inner-wrapper'>
+        <div className="cookie-agreement-title">
+          <h2>Welcome To </h2>
+          <img
+            src="/static/media/brand-text.320ee44d.png"
+            alt="ClassTranscribe" 
+          />
+        </div>
+          
+        <p id="cookie-agreement-desc">
+          By continuing you agree to the <Link to='/policy/acceptableUse'> acceptable_use policy</Link> and <Link to='/policy/cookie'>cookie policy</Link>.
+        </p>
+        <div id="cookie-agreement-btn-grp">
+          <div
+            className="cookie-agreement-btn"
+            onClick={handleAcceptLoginClick}
+            role='button'
+            tabIndex='0'
+          >
```

Comment:
- I think you can use `ButtonBase` component of material-ui for this, check out [example](https://material-ui.com/components/buttons/#complex-buttons) and [ButtonBase API](https://material-ui.com/api/button-base/).
- Also, when a button is active (click and hold), its position changes a little bit. it will be better if this can be fixed